E2a-TRAS/SB01/README.md

```diff
@@ -0,0 +1,156 @@
+# Test instructions
+
+| Documentation Code   | Test Title                                 | Exchange Code | Test Code | Author          | Data Owner | Version | Date       |
+|----------------------|--------------------------------------------|---------------|-----------| ----------------|------------|---------|------------|
+| IFC4.3AbRV_E2a_SB01  | (RFI) Track sub-structure for single track | E2a (TRAS)    | SB01      | Evandro Alfieri | RFI        | 1.0     | DD.MM.YYYY |
+
+
+
+
+## Summary (Intent)
+
+This test case addresses the **export** of the required IFC entities for the exchange of **track sub-structure** information.
+
+This test case focuses only on track sub-structure (red in picture below). For the track super-structure part (green in picture below), refer to test case **IFC4.3AbRV_E2a_SP01**, of which SB01 is a pre-condition.
+
+Snippet:
+<img src="./Dataset/TrackCrossSection_snippet.png" width="500"/>
+
+- Refer to [Test Case Imports](#Test-Case-Imports) to know the prerequisites for the present test.
+
+- The [Expected Results](#Expected-Results) section lists the material that will be used to assess the fulfilment of capabilities.
+
+- :zap: This is a test-driven process: refer to the [Validation Criteria](#Validation-Criteria) to understand what is required by the test :zap:
+
+
+
+
+
+## Itemised Roots
+:construction: **under construction, check CT for product shape** :construction:
+
+The Test instruction addresses the import and export of the following IFC Entities & Concept Templates:
+
+<details><summary>IFC Entities</summary>
+
+These entities represent a test-specific subset of the wider AbRV_E2a exchange and the overall AbRV MVD. **The scope of the test shall not be used as a definitive scope of the exchange, or of the MVD**
+
+- Products:
+   1. IfcCourse / IfcCourseType
```

Comment:
What is IfcCourseType supposed to do here, does it capture any information?

Reply:
later, in the pset test